code/unclassified/leap_year/leap_year.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+using namespace std;
```

Comment:
move using declaration to main function, to avoid name conflict.

Reply:
If i try this then I had to add same line at 2 places i.e. Inmainfunction as well as in findLeapYears function